src/utils/config.rs

```diff
@@ -0,0 +1,7 @@
+pub const IPFS_API_PORT:u16 = 4869;
+pub const IPFS_RETRY_ATTEMPTS:u16 = 4;
+pub const IPFS_ADDR:&str = "127.0.0.1";
+pub const IPFS_EXE:&str = "ipfs";
+pub const HTTP_BOUNDARY:&str = "------------------------I_am_a_boundary_123A123";
+pub const IPFS_BOOT_TIME_OUT:u16 = 45;//In seconds
```

Comment:
```suggestion
pub const IPFS_BOOT_TIME_OUT:u16 = 45; // In seconds
```

Related to the above comment about `//` comments. I'd propose we add a space before the `//` and a space after. This makes the comments easier to read and follows the style used by tokio. For example: https://github.com/tokio-rs/tokio/blob/c6552c5680fa14105547cfbbc26b26d67197b64e/tokio/src/runtime/context.rs#L133